client/app/utils/enums.ts

```diff
@@ -46,7 +46,6 @@ export enum UserOperatorStatus {
 
 // user statuses
 export enum Status {
-  MYSELF = "Myself",
```

Comment:
I would have liked to change this to MYSELF = "Approved", but we use the status text to decide whether or not to show approve/reject buttons, more details below